src/views/Dashboard/index.tsx

```diff
@@ -49,10 +51,13 @@ export const Dashboard: React.FC<{}> = () => {
       balances?.data?.filter(asset => !asset.balance.isZero()) ?? []
     ).map(a => ({ ...a, backingReserve: a.reserve }));
     return assets.map(asset => {
-      return asset.backingReserve && asset.backingReserve.symbol === "WXDAI"
+      return asset.backingReserve.symbol === nativeSymbols.wrappednative
         ? {
             ...asset,
-            backingReserve: { ...asset.backingReserve, symbol: "XDAI" },
+            backingReserve: {
+              ...asset.backingReserve,
+              symbol: nativeSymbols?.native,
```

Comment:
This produces an object even when the symbol is undefined; did you want to not produce an object *at all* in that case? If so, the `?` on `nativeSymbols?.native` must be made unnecessary.